library/network/src/modules/SuSEFirewall.rb

```diff
@@ -1320,6 +1320,54 @@ def SetServicesForZones(services_ids, firewall_zones, new_status)
       nil
     end
 
+    # Function returns actual state of Masquerading support.
+    # In FirewallD, masquerade is enabled per-zone so this
+    # function has to iterate through all the zones and return
+    # a collective boolean value. This API is probably not very
```

Comment:
I am not sure. Because the old API does not quite match the new one, whatever decision is made is not going to be 100% correct. Nevertheless, I am happy to change the code to treat the "internal" as default zone.